src/Components/BeastImage.js

```diff
@@ -0,0 +1,64 @@
+//import Image from 'react-bootstrap/Image';
+import { Component } from 'react';
```

Comment:
Hi there, you will want to change the import statement:

`import React , {Component} from 'react'`